TripHouse Project/index.html

```diff
@@ -7,9 +7,106 @@
     <meta name="author" content="Bega Rakhmankulov "/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <link rel="stylesheet" href="../styles/style.css"/>
-    <title>lesson 5</title>
+    <title>lesson 6</title>
 </head>
 <body>
+<section class="main-page">
+    <header class="header-content">
+        <a class="header-logo" href="#">
+        <img src="../images/triphouse-logo.png" alt="TripHouse-logo">
+        </a>
+        <nav class="header-right-content">
+            <ul class="nav-content">
+                <li>
+                    <a class="nav-content links" href="#">Stays</a>
+                </li>
+                <li>
+                    <a class="nav-content links" href="#">Attractions</a>
+                </li>
+            </ul>
+            <ul class="nav-content icons">
+                <li>
+                    <img
+                            class="icon-picture night-theme"
+                            src="../images/night-icon.svg"
+                            alt="Night-Theme"
+                    />
+                </li>
+                <li>
+                    <img
+                            class="icon-picture account-icon"
+                            src="../images/account-icon.svg"
+                            alt="Account-icon"
+                    />
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <div class="main-text">
+        Discover stays <br/>
+        to live, work or just relax
+    </div>
+    <form action="/" method="get">
+        <div class="labels-container">
```

Comment:
This is a bad idea to use labels and inputs in diffrent wrappers
You did
```
<>Title Title</>
<>Input Input</>
```

I recommend
```
<>Title Input</>
<>Title Input</>
```


It will be really difficult to do a mobile version in the future with such a layout

Anyway you can see a video-tutorial about styling of this block